scripts/clean_fcctr.sh

```diff
@@ -47,9 +47,18 @@ for d in `find /var/lib/cni/ -mindepth 1 -maxdepth 1  -type d | grep -v networks
     sudo rm -rf $d
 done
 
-echo Removing devmapper devices
-for de in `sudo dmsetup ls| cut -f1|grep snap`; do sudo dmsetup remove $de && echo Removed $de; done
-sudo dmsetup remove fc-dev-thinpool
+CONTAINERID=$(basename $(cat /proc/1/cpuset))
+
+if [ 64 -eq ${#CONTAINERID} ]; then
```

Comment:
what is 64? in this and in several other files.

Reply:
Docker container ID is always 64 characters long. If you run this natively, the return value is "/".